src/isolate.rs

```diff
@@ -45,6 +46,13 @@ impl Isolate {
             IsolateHolder::Borrowed(p) => p,
         }
     }
+
+    pub fn current_context(&self) -> context::Context {
+        unsafe { 
```

Comment:
I think this might segfault if there is no context scope bound, so if I call `Isolate::new().current_context()` for example. Can't confirm though since I'm on the train without a laptop.

If that's the case, I see the options:

  -  Pass in the current context in `*CallbackInfo`
  - Track context bound and unbound isolates separately
  - Do some runtime check here and panic or something